CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ option(LIBMOTIONCAPTURE_ENABLE_QUALISYS "Enable Qualisys" ON)
 option(LIBMOTIONCAPTURE_ENABLE_OPTITRACK "Enable Optitrack" ON)
 option(LIBMOTIONCAPTURE_ENABLE_OPTITRACK_CLOSED_SOURCE "Enable Optitrack (Closed Source)" OFF)
 option(LIBMOTIONCAPTURE_ENABLE_VICON "Enable Vicon" ON)
+option(LIBMOTIONCAPTURE_ENABLE_NOKOV "Enable Nokov" ON)
```

Comment:
This should be disabled by default, since the user needs to manually find and copy the SDK

Reply:
Thank you for reminding.